### DISCUSSION TITLE
Federal requirements for banner

### DISCUSSION THREAD

**Discussion by sarinapadilla** [NCIOCPL/ncids#1602] [NCIDS Launch]
From our review with the BRIDG team, we noted the lack of documentation on the Banner component page concerning the federal requirement of a banner.

**Comment by blilianyu** [comment]
On the Banner component page in the intro, we can add a sentence about how it is federal requirement due to the 21st Century IDEA Act.

cc: @monika-jaeger

**Comment by laurelthrash** [comment]
Discussed during 04/18 UX meeting. @adrianacastaneda to draft suggested language and page placement (and add to this discussion). We will align internally then run by Anna prior to implementing.

**Comment by adrianacastaneda** [comment]
Based on discussion on the[ Discussions ticket](https://github.com/orgs/NCIOCPL/projects/16/views/15?pane=issue&itemId=57902221), it looks like we're moving forward with option 1! @laurelthrash 

option 1: A banner used to identify official websites of government organizations in the United States. It allows users to know if a website is both official and secure. This component is required for all NCI informational websites and applications.


cc: @blilianyu @lburack

**Comment by laurelthrash** [comment]
Received! Since this issue was reported by the internal team, as opposed to the federal team, during review with BRIDG and since it is not a major change, let's proceed directly with making the update. 

@andyvanavery31 @sarinapadilla are you available to please create a CR ticket for this and add to the Redesign board?

**Reply by blilianyu** [reply]
I've created the CR #1670 @sarinapadilla @andyvanavery31 @laurelthrash
